docs/security/threat-modelling.md

```diff
@@ -0,0 +1,26 @@
+# Threat Modeling
+
+Threat modeling is an effective way to help secure your systems, applications, networks, and services. It's an systematic approach that identifies potential threats and recommendations to help reduce risk and meet security objectives earlier in the development lifecycle.
+
+## Threat Modeling Phases
+
+1. *Diagram*  
```

Comment:
Just a suggestion: Maybe we can add some project example of this flow?

Reply:
Add the link to example document.

> Just a suggestion: Maybe we can add some project example of this flow?

Adding link to example document